app/uk/gov/hmrc/cdsimportsddsfrontend/controllers/model/DeclarationViewModel.scala

```diff
@@ -87,4 +88,24 @@ case class DeclarationViewModel(
       obligationGuarantee = Some(miscellaneousViewModel.toObligationGuarantee)
     )
   }
+
+  private def combineAdditionalDocsAndWriteOffs(
+     additionalDocuments: Seq[AdditionalDocumentViewModel],
+     writeOffs: Seq[WriteOffViewModel]): Seq[AdditionalDocument] = {
+    additionalDocuments.zip(writeOffs).map( docsAndWriteOffPair => toAdditionalDocument(docsAndWriteOffPair._1, docsAndWriteOffPair._2))
+  }
```

Comment:
could this be rewritten somehow like `(additionalDocument, writeOff) => ...` to avoid the `_1` and `_2` ugliness?

Reply:
Yeah good point, done.

Reply:
just do this:
additionalDocuments.zip(writeOffs).map{case (doc, writeoff) => toAdditionalDocument(doc, writeoff)}